src/lib.rs

```diff
@@ -412,4 +451,31 @@ mod tests {
         assert_eq!(person.downcast_mut::<Chris>(), None);
         assert!(person.downcast::<Chris>().err().is_some());
     }
+
+    #[test]
+    fn parameterised() {
```

Comment:
You have two spellings in here, `Parameterized` and `parameterised`. Pick one or the other—I don’t mind which; the latter is correct in my language (en_AU) but I’m used to programming in both en_AU and en_US and haven’t made any decision here…